eng/Version.Details.xml

```diff
@@ -31,13 +31,13 @@
       <Uri>https://github.com/dotnet/arcade</Uri>
       <Sha>70831f0d126fe88b81d7dc8de11358e17a5ce364</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.DotNet.Build.Tasks.Feed" Version="9.0.0-beta.24408.2">
-      <Uri>https://github.com/dotnet/arcade</Uri>
-      <Sha>60ae233c3d77f11c5fdb53e570b64d503b13ba59</Sha>
+    <Dependency Name="Microsoft.DotNet.Build.Tasks.Feed" Version="10.0.0-beta.25223.119">
+      <Uri>https://github.com/dotnet/dotnet</Uri>
+      <Sha>721dc7a2a59416b21fc49447d264009d708d6000</Sha>
     </Dependency>
-    <Dependency Name="Microsoft.TemplateEngine.Authoring.Tasks" Version="9.0.201">
-      <Uri>https://github.com/dotnet/templating</Uri>
-      <Sha>006b799c44481a0870d4cbe754e6aae9d1236708</Sha>
+    <Dependency Name="Microsoft.TemplateEngine.Authoring.Tasks" Version="10.0.100-preview.5.25223.119">
+      <Uri>https://github.com/dotnet/dotnet</Uri>
+      <Sha>721dc7a2a59416b21fc49447d264009d708d6000</Sha>
```

Comment:
@pjcollins is there any concern updating these? Or do some of our yaml templates use a different dotnet/arcade version anyway?

Reply:
We should probably add these to the excluded asset list so there's no major version changes

Reply:
@dkurepa do you know if there a way to ignore certain dependencies in a subscription? This would help us update the downlevel .NET 8/9 packs on main more effectively.

For example, I was trying to track `Microsoft.NET.Workload.Mono.ToolChain.Current.Manifest-8.0.100` here from dotnet/runtime/release/8.0:

https://github.com/dotnet/android/blob/1d799637ffd89ee1f409677dcb5875542bea1f09/eng/Version.Details.xml#L27-L31

But I bet it will update on the next dotnet/runtime/release/9.0 update.

I think this PR is probably fine as-is, I just wanted to check about the two dependencies before merging.

Reply:
yes! The `--excluded-assets` flag should help you with this. So you can just run `darc update-subscription --id 58e8279d-a615-4b62-adba-499b6c5c9d3e --excluded-assets <list of assets separated by ;>`. Note that this command will always update the whole list of excluded assets, so always list all of them

Reply:
For the subscription:
```
https://github.com/dotnet/runtime (.NET 9) ==> 'https://github.com/dotnet/android' ('release/9.0.1xx')
  - Id: 5321cb53-127e-4a94-91ac-29929edb8b1c
```

I tried:
```
> darc update-subscription --id 5321cb53-127e-4a94-91ac-29929edb8b1c --excluded-assets Microsoft.NET.Workload.Mono.ToolChain.Current.Manifest-8.0.100
Asset exclusion only works for source-enabled subscriptions
```

Reply:
> @pjcollins is there any concern updating these? Or do some of our yaml templates use a different dotnet/arcade version anyway?

I don't think this will be an issue for us, and it's probably for the best for these to be updated more regularly. Our MSI generation tasks still are pinned on a specific arcade version in a separate repo.

Reply:
@jonathanpeppers tracking `Microsoft.NET.Workload.Mono.ToolChain.Current.Manifest-8.0.100` makes no sense in this case since that is produced in a completely different channel.

Let's talk about the use case you have for this separately